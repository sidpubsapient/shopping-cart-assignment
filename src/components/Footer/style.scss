.footer {
    align-items: center;
    background-color: #eee;
    bottom: 0;
    display: flex;
    height: 4rem;
    justify-content: center;
    font-size: 1.2rem;
}
