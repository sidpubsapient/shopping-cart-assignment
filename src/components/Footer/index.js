import React from "react";
import Text from "../Text";

import "./style.scss";

const Footer = () => (
    <footer className="footer">
        <Text>
            Copyright &copy; 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd
        </Text>
    </footer>
);

export default Footer;
