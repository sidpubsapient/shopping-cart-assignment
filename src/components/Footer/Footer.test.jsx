import React from "react";
import { render } from "../../../tests/test-utils";

import Footer from "./index";

it("should take a snapshot", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment(<Footer />)).toMatchSnapshot();
});
