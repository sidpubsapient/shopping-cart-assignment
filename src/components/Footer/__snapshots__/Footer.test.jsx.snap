// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <span
      aria-hidden="false"
      aria-label=""
      class="span "
    >
      Copyright Â© 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd
    </span>
  </footer>
</DocumentFragment>
`;
